app/templates/components/ui-table/cell/cell-simple-date.hbs

```diff
@@ -1,6 +1,6 @@
-{{#if record.submittedAt}}
+{{#if (get record column.propertyName)}}
   <span>
-    {{moment-format record.submittedAt 'MMMM DD, YYYY - HH:mm A'}}
+    {{moment-format (get record column.propertyName) 'MMMM DD, YYYY - HH:mm A'}}
```

Comment:
Allow customization of the display format if needed.

Reply:
@niranjan94 customization in what sense?
Something like the user can decide the format of the date or on the admin level?

Reply:
Customization on the code level. Allow passing in the date format needed in the the column definition